modules/uhttp/filters.go

```diff
@@ -65,15 +65,15 @@ func (f tracingServerFilter) Apply(ctx context.Context, w http.ResponseWriter, r
 	carrier := opentracing.HTTPHeadersCarrier(r.Header)
 	spanCtx, err := opentracing.GlobalTracer().Extract(opentracing.HTTPHeaders, carrier)
 	if err != nil && err != opentracing.ErrSpanContextNotFound {
-		fx.Logger(ctx).Warn("Malformed inbound tracing context: ", "error", err.Error())
+		ulog.Logger(ctx).Warn("Malformed inbound tracing context: ", "error", err.Error())
```

Comment:
I'll have a follow up PR when host.Logger() is removed, since this PR is only refactoring current setup.